app/components/resource_button.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ResourceButton < ApplicationComponent
+  attr_reader :resource, :policy
+
+  def initialize(resource:, policy: nil)
+    @resource = resource
+    @policy = policy
+  end
+
+  def method
+    nil
```

Comment:
use `def method; end`

`nil` is implied. This is typically the _only_ place we're you'll find one-line method invocations.